user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg

```diff
@@ -0,0 +1,65 @@
+#-------------------------------------------------#
+#### LDO Picobilical Frame PCB A1.3 definition ####
+#-------------------------------------------------#
+
+[mcu toolhead]
+##--------------------------------------------------------------------
+serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+restart_method: command
+##--------------------------------------------------------------------
+
+# If you want to override the wiring of the LDO Picobilical, keep in mind that this
+# board is defined using the "toolhead" name. So you should use "pin: toolhead:PIN_NAME"
+# in your own overrides.cfg files.
+
+[include config/mcu_definitions/toolhead/LDO_Picobilical_vA1.3.cfg] # Do not remove this line
+[board_pins picobilical_mcu]
+mcu: toolhead
+aliases:
+    X_STOP=MCU_XES ,
+
+    RUNOUT_SENSOR=MCU_FS ,
+
+    E_HEATER=MCU_HE0 , E_TEMPERATURE=MCU_TH0 ,
+
+    PART_FAN=MCU_PCF , E_FAN=MCU_HEF ,
+
+    PB_PCB_TEMP=MCU_CT ,
+
+    # If you want to use both of the independent neopixel ports,
+    # comment the below line and uncomment the left/right neopixel line
+    # NOTE: YOU WILL NOT BE ABLE TO USE THE STOCK KLIPPAIN
+    # MACROS WITH THE DUAL NEOPIXEL CONFIGURATION!
+    LIGHT_NEOPIXEL=MCU_NEOPIXEL_1
+    # RIGHT_NEOPIXEL=MCU_NEOPIXEL_1 , LEFT_NEOPIXEL=MCU_NEOPIXEL_2
+
+#----------------------------------------#
+#  LDO Picobilical vA1.3 pins remapping  #
+#----------------------------------------#
+
+# These pins overrides are automatically added when you select a
+# toolhead MCU during the installation process. They should provide a
+# good base to work with. Feel free to adapt to your board if needed!
+
+[extruder]
+heater_pin: toolhead:E_HEATER
+sensor_pin: toolhead:E_TEMPERATURE
+pullup_resistor: 2200
+
+[fan]
+pin: toolhead:PART_FAN
+
+[heater_fan hotend_fan]
+pin: toolhead:E_FAN
+
+## Uncomment the following line if not using sensorless homing
+## and having the X endstop plugged to the toolhead MCU
+# [stepper_x]
+# endstop_pin: ^toolhead:X_STOP
+
+[temperature_sensor picobilical_temp]
+sensor_type: CMFB103F3950FANT
+sensor_pin: PB_PCB_TEMP
+pullup_resistor: 2200
+min_temp: 0
+max_temp: 100
```

Comment:
Can you add two blank new lines at the end of the file to allow it to be injected to the mcu.cfg user file easily?

Reply:
Done, closing